src/main/webapp/i18n/de/participantScores.json

```
@@ -1,11 +1,11 @@
 {
     "artemisApp": {
         "participantScores": {
-            "pageTitle": "Teilnehmer Ergebnisse",
+            "pageTitle": "Teilnahme Ergebnisse",
```

```suggestion
            "pageTitle": "Teilnahmeergebnisse",
```
Or at least Teilnahme-Ergebnisse